google/resource_cloudfunctions_function.go

```diff
@@ -530,11 +530,17 @@ func expandEventTrigger(configured []interface{}, project string) *cloudfunction
 		shape = "projects/%s/buckets/%s"
 	case strings.HasPrefix(eventType, "google.pubsub.topic."):
 		shape = "projects/%s/topics/%s"
+	case strings.HasPrefix(eventType, "google.firestore.document."):
```

Comment:
Are there any triggers using this shape? I only see the other form [in the docs](https://cloud.google.com/functions/docs/calling/cloud-firestore)

Reply:
You're right, I for some reason thought there were a google.firestore.document series of event types. I have removed this case.